boards/samr21-xpro/Makefile.dep

```diff
@@ -5,3 +5,7 @@ endif
 ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_gpio
 endif
+
+ifneq (,$(filter usbdev,$(USEMODULE)))
+  USEMODULE += sam_usb
```

Comment:
Why do you enable this feature for SAMR21 board only ?
You can put it directly in `cpu/sam0_common/Makefile.dep` instead.

Reply:
> Why do you enable this feature for SAMR21 board only ?

My idea here was to only enable the the feature for boards that expose the USB port.